changelogs/unreleased/required-field-in-debt-recovery-method-line.yml

```diff
@@ -0,0 +1,3 @@
+---
+title: "DEBTRECOVERYMETHODLINE : make debtRecoveryLevel field required in debt-recovery-method-line."
```

Comment:
keep it short and simple
like, 
title: "DEBTRECOVERYMETHODLINE : make debtRecoveryLevel field required"